pom.xml

```diff
@@ -6,14 +6,14 @@
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-spanner-hibernate</artifactId>
   <packaging>pom</packaging>
-  <version>1.1.0-SNAPSHOT</version>
+  <version>1.2.0-SNAPSHOT</version>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <hibernate.version>5.4.9.Final</hibernate.version>
-    <spanner-jdbc-driver.version>1.12.0</spanner-jdbc-driver.version>
+    <spanner-jdbc-driver.version>1.13.0</spanner-jdbc-driver.version>
```

Comment:
Not important, but probably should've done this update before the release?

Reply:
Ah good point.